pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -156,8 +157,14 @@ func (a *AgentClusterInstall) Generate(dependencies asset.Parents) error {
 			},
 		}
 
-		if installConfig.Config.Platform.Name() == none.Name {
-			logrus.Debugf("Setting UserManagedNetworking to true for %s platform", none.Name)
+		if installConfig.Config.Platform.Name() == external.Name {
```

Comment:
This works if the user started with an install-config, but if they started with ZTP manifests it won't:

```suggestion
		if agentClusterInstall.Spec.PlatformType == hiveext.ExternalPlatformType {
```